### ISSUE TITLE
Need webpack bundling

### ISSUE THREAD

**Issue by podfeet** [bartificer/xkpasswd-js#32]
I'm working in a branch on my fork and want to be able to see the running web app in a GitHub page. Webpack is running but not configured to do this. (poor title on my part but I can't edit it to better reflect the problem.)

Also - I'd like to be assigned this issue.

**Comment by hepabolu** [comment]
It’s not webpack that does this. It’s the GitHub action. It is triggered by a commit to main. You might want to go in and change the branch name in the workflow yaml.

**Comment by podfeet** [comment]
I'm afraid I don't know what any of this means - and I'm pretty sure I can't mess with the workflow yaml. I've unassigned myself.

**Comment by hepabolu** [comment]
@podfeet has this issue been resolved? If so, can you close it?

**Comment by podfeet** [comment]
I wrote last week about this because I don't understand your answer. If this is to be a standalone web app that's transportable, I thought it needed to be a Webpack bundle. Your answer was about workflow yaml and I don't know what that is or what it has to do with Webpack bundling. 

Maybe we can deliver this as a standalone bundle without Webpack but I don't know how to do that or what yaml is or has to do with this. Hence removing myself from the assignee.

**Comment by hepabolu** [comment]
This is not a standalone app, like the Slack desktop app. 

Maybe this confusion is caused by my definitions: I consider a place with mainly information, i.e. posts and pages, to be a website, while a web app provides a service like this project (XKPasswd) and regex101.com. An app that runs on your desktop, without opening a browser (yes, technically Electron does just that: opening a dedicated browser window), is just that: an app.

Webpack is used for gathering all the necessary files and their dependencies to make the website go. It places all files in the `dist` directory which is then uploaded to GitHub pages using the GitHub workflow.

From your initial post here I assumed that you wanted to run your own GitHub Pages version in your own fork, based on the branch you preferred. That is what I had you do in our video call.
